geniza/corpus/models.py

```diff
@@ -925,6 +925,65 @@ def title(self):
         """Short title for identifying the document, e.g. via search."""
         return f"{self.doctype or _('Unknown type')}: {self.shelfmark_display or '??'}"
 
+    def dating_range(self):
+        """
+        Return the start and end of the document's possible date range, as PartialDate objects,
+        including standardized document dates and inferred Datings, if any exist.
+        """
+        # it is unlikely, but technically possible, that a document could have both on-document
+        # dates and inferred datings, so find the min and max out of all of them.
+        start_date = self.start_date if self.start_date else None
+        end_date = self.end_date if self.end_date else None
+        dating_range = [start_date, end_date]
```

Comment:
what are the values if they aren't set?

think this can be simpler:
```suggestion
        dating_range = [self.start_date or None, self.end_date or None]
```